roles/lead_software_engineer.md

```diff
@@ -91,6 +91,8 @@ Compensating you fairly:
 
 ## Salary
 
+The salary for this role is location dependant:
+
 - Bristol: £63,900-£81,000
 - Manchester: £63,900-£81,000
 - London: £71,000-£90,000
```

Comment:
Could Bristol and Manchester be together where they are the same?
```suggestion
- Bristol & Manchester: £63,900-£81,000
- London: £71,000-£90,000
```

Reply:
No guarantee they'll stay the same, or that we won't add more locations. Seems reasonable to me to duplicate for clarity?